scheduler/scheduler.cpp

```diff
@@ -908,7 +1054,7 @@ static time_t prune_servers()
     return min_time;
 }
 
-static bool empty_queue()
+static bool empty_queue(SchedulerAlgorithmName scheduler)
```

Comment:
calling this variable scheduler doesn't make sense.  Can it be renamed to algorithm?  Or even schedulerAlgroithm

Reply:
Yes, I think this is one I missed in my search-and-replace. I'll fix it up in the next commit.